hack/tests/e2e-helm.sh

```diff
@@ -125,12 +125,16 @@ if echo $log | grep -q "failed to generate RBAC rules"; then
 fi
 
 pushd nginx-operator
-sed -i 's|\(FROM quay.io/operator-framework/helm-operator\)\(:.*\)\?|\1:dev|g' build/Dockerfile
+if [[ "$OSTYPE" == "darwin"* ]]; then
+  sed -i " " 's|\(FROM quay.io/operator-framework/helm-operator\)\(:.*\)\?|\1:dev|g' build/Dockerfile
+else
+  sed -i 's|\(FROM quay.io/operator-framework/helm-operator\)\(:.*\)\?|\1:dev|g' build/Dockerfile
+fi
```

Comment:
Can this be combined into one command, similar to the others?

Reply:
Note that it is in WIP and I am doing it step by step because I am trying to make the regex work with your suggestion so far. See that the current changes are not passing:

> +sed -i.bak -E -e 's|{{ REPLACE_IMAGE }}|quay.io/example/memcached-operator:v0.0.2|g' deploy/operator.yaml
> sed: -e expression #1, char 65: Invalid preceding regular expression
> +rm -rf /tmp/tmp.fNuNyKjuVW

The error is with: https://github.com/operator-framework/operator-sdk/pull/2260/files#diff-a8c10525e5714b373fd8c481bd9feb85R156